find_posts.py

```diff
@@ -27,9 +27,11 @@
 argparser.add_argument('--remember-users-for-hours', required=False, type=int, default=24*7, help="How long to remember users that you aren't following for, before trying to backfill them again.")
 argparser.add_argument('--http-timeout', required = False, type=int, default=5, help="The timeout for any HTTP requests to your own, or other instances.")
 argparser.add_argument('--lock-hours', required = False, type=int, default=24, help="The lock timeout in hours.")
+argparser.add_argument('--lock-file', required = False, default=None, help="Location of the lock file")
```

Comment:
Python is eagerly evaluated, and `arguments` doesn't exist yet, so that won't work. And I don't see any easy way to add a lazily-evaluated `default` here.

Reply:
Thanks. I thought you'd say that, hence putting it as a question. Shame, but not much we can do